Automattic/wp-calypso
# Add a4a overview skeleton

Related to # https://github.com/Automattic/jetpack-genesis/issues/297

## Proposed Changes
Create a skeleton for easier development split in the future. Proposed organisation of the code as follows:

<img width="463" alt="Screenshot 2024-03-18 at 12 31 09 PM" src="https://github.com/Automattic/wp-calypso/assets/60262784/b113ddfb-9171-41a8-9596-34030ac98211">

I've split every section in their own files, but we could have all stack up in one `index.tsx` if we want to.

## Testing Instructions



* On the existing branch navigate to `/overview` page (eg. http://agencies.localhost:3000/overview).
* See placeholders for each section of main menu and sidebar

<img width="1078" alt="Screenshot 2024-03-18 at 12 38 49 PM" src="https://github.com/Automattic/wp-calypso/assets/60262784/6fd01fcd-5819-4f4c-9f85-1f3551b67875">


## Pre-merge Checklist
